frontend/src/routes/index.tsx

```diff
@@ -56,7 +60,7 @@ export default function Router() {
       element: <Page404 />,
     },
     { path: '/', element: <Navigate to="/dashboard?disableNotification=true" replace /> },
```

Comment:
Oops I accidentally added this in my commit. Can help to remove ty.

Reply:
Sure!